_data/translation-management-systems.yaml

```diff
@@ -0,0 +1,669 @@
+
+  features:
+    - name: Local machine translation engine
+      description: Users can use their local machine translation in the tool
+
+    - name: Fuzzy repair with machine translation
+      description: Content that changes in the new source is fixed in target with machine translation
+      link: https://github.com/transducens/OmegaT-FuzzyMatchRepair-Plugin
+
+    - name: Suggestions from multiple machine translation engines
+      description: Users can see segment-level suggestions from several machine translation systems at the same time
+
+- id: passolo
+  name: Passolo
+  description: Software localization tool
+  url: https://www.trados.com/products/passolo/
+  cat: true
+  documentation: https://docs.rws.com/914743/262054/passolo-help/welcome-to-the-------------passolo-documentation
+  machine_translation_integrations:
+    - service: google
+      custom: true
+
+    - service: language-weaver
+      custom: true
+
+    - service: microsoft
+      custom: true
+
+  machine_translation_plugins:
+    - service: deepl
+      link: https://appstore.rws.com/language/app/deepl-mt-provider-for-passolo/1088/
+
+  features:
+    - name: Suggestions from multiple machine translation engines
+      description: Users can see segment-level suggestions from several machine translation systems at the same time
+
+- id: smartcat
+  name: Smartcat
+  description: Cloud-based translation and localization platform
+  url: https://www.smartcat.ai
+  cat: true
+  documentation: https://help.smartcat.com/hc/en-us/
+  machine_translation_integrations:
+    - service: amazon
+      custom: true
+
+    - service: baidu
+
+    - service: deepl
+
+    - service: google
+      custom: true
+      glossary: true
+
+    - service: intento
+      glossary: true
+
+    - service: microsoft
+      custom: true
+
+    - service: modernmt
+      custom: true
+
+    - service: yandex
+
+    machine_translation_plugins:
+    - service: globalese
+      custom: true
+
+  features:
+    - name: Suggestions from multiple machine translation engines
+      description: Users can see segment-level suggestions from several machine translation systems at the same time
+
+- id: smartling
+  name: Smartling
+  description: Online translation management and localization software
+  url: https://www.smartling.com
+  cat: true
+  documentation: https://help.smartling.com/hc/en-us
+  machine_translation_integrations:
+    - service: amazon
+
+    - service: deepl
+
+    - service: google
+      custom: true
+      glossary: true
+      link: https://help.smartling.com/hc/en-us/articles/1260805168070-Translating-with-Google-AutoML
+
+    - service: intento
+
+    - service: microsoft
+      custom: true
+
+    - service: promt
+
+    - service: systran
+
+    - service: watson
+      custom: true
+
+  features:
+    - name: Local machine translation engine
+      description: Users can use their local machine translation in the tool
+
+- id: trados
+  name: Trados Studio
```

Comment:
This isn't a TMS though.

Reply:
If we want to have CATs like Trados and OmegaT here, then we should rename this file to integrations.yml.

And then we could have fields to say whether it's a CAT, TMS or both, although I don't think it's that important, since that's not our mission, and there are other resources on that.

Reply:
Yeah, I think it's important to have Trados or OmegaT here. I'll rename the file.
The thing with Trados is tricky. Many RWS products only support RWS MT, but all other are available through Trados and the RWS AppStore. For instance, when researching tools for Passolo, RWS support always suggests exporting the file for Trados and pretranslating there.